core/Profiler.php

```diff
@@ -209,13 +209,9 @@ public static function setupProfilerXHProf($mainRun = false, $setupDuringTrackin
             return;
         }
 
-        $xhProfPath = PIWIK_INCLUDE_PATH . '/vendor/facebook/xhprof/extension/modules/xhprof.so';
-        if (!file_exists($xhProfPath)) {
-            throw new Exception("Cannot find xhprof, run 'composer install --dev' and build the extension.");
-        }
-
         if (!function_exists('xhprof_enable')) {
-            throw new Exception("Cannot find xhprof_enable, make sure to add 'extension=$xhProfPath' to your php.ini.");
+            $xhProfPath = PIWIK_INCLUDE_PATH . '/vendor/facebook/xhprof/extension/modules/xhprof.so';
```

Comment:
Here should be another condition with file_exists, so the order would be:
- try to use global installed extension
- try to check if local extension exists
- throw an exception